Foundation/Operation.swift

```diff
@@ -168,6 +168,21 @@ open class Operation : NSObject {
 #endif
         _ready = true
     }
+
+    internal func _checkReadyness() -> Bool {
+        if _ready {
```

Comment:
If a dependency is added to this operation, shouldn't it reset its readiness?

Reply:
Good question
But now if `_ready` is set then operation will be immediately executed. So this if have no many sense.
Maybe this `if` should be removed to not confuse. And proper isReady mangement should be in different PR.